src/components/TableComponent/TableComponent.test.jsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import TableComponent from './index';
+
+describe('CourseTable', () => {
+  const mockFormat = data => data.map(datum => ({
+    ...datum,
+  }));
+  const mockColumns = [
+    {
+      label: 'Col 1',
+      key: 'col1',
+      columnSortable: true,
+    },
+    {
+      label: 'Col 2',
+      key: 'col2',
+      columnSortable: false,
+    },
+  ];
+  const mockFetch = () => [
+    {
+      col1: 'Test11',
+      col2: 'Test12',
+    },
+    {
+      col1: 'Test21',
+      col2: 'Test22',
+    },
+  ];
+
+  it('shows a table', () => {
+    const component = shallow(<TableComponent
+      id="test"
+      className="test"
+      fetchMethod={mockFetch}
+      columns={mockColumns}
+      formatData={mockFormat}
+      tableSortable
+      paginateTable={() => true}
+      sortTable={() => true}
+      clearTable={() => true}
+      location={{ search: '?page=1&ordering=key' }}
+    />);
+    expect(component).toMatchSnapshot();
```

Comment:
I am still new to snapshots, but should we also check for its presence in the DOM?  My understanding is that the snapshot will only ensure we do not change the structure of the DOM unexpectedly.